app/abcs/abc_download.py

```
@@ -46,12 +58,65 @@ def check_if_folder_exists(self) -> Path:
+                .cast(pl.String)
+                for col in pl_file.collect_schema().names()
+                ]))
+
+        pl_file.collect().write_parquet(tmp / f"{rename}.parquet", compression='lz4')
+        progress.update_task(_csv_task, "Complete")
+        # Clean up original CSV file
+        (tmp / file_name).unlink()
+
+    @classmethod
+    @abc.abstractmethod
+    def download(cls, overwrite: bool, read_from_temp: bool) -> FileDownloaderABC:
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)